sw/device/tests/sleep_pin_mio_dio_val_test.c

```diff
@@ -34,11 +34,18 @@ OTTF_DEFINE_TEST_CONFIG();
  * the expected and measured values.
  */
 
+#if defined(OPENTITAN_IS_EARLGREY)
 enum { kNumOptOut = 2 };
 static const dt_pad_t kOptOut[] = {
     kDtPadSpiDeviceSck,
     kDtPadSpiDeviceCsb,
 };
+#elif defined(OPENTITAN_IS_DARJEELING)
+enum { kNumOptOut = 0 };
```

Comment:
Nit: how would you feel about this being a Bool so that it doesn't have to be kept in sync with the `ARRAYSIZE`?